src/edu/stanford/nlp/parser/lexparser/EvaluateTreebank.java

```diff
@@ -391,7 +392,7 @@ else if(pwFileOut != null) {
           double denom = ArrayMath.logSum(logScores);
           for (double logScore : logScores) {
             double logPr = logScore - denom;
-            entropy += Math.exp(logPr) * (logPr / Math.log(2));
+            entropy += FastMath.exp(logPr) * (logPr / FastMath.log(2));
```

Comment:
It should likely use the same constant `LN_TO_LOG2=1/log(2)` as above, with multiplication rather than division. The numerical differences between division and multiplication with the inverse should be negligible here, since `log(2)` does not have an exact representation in floating point.